var/external/bufr/bufrlib.h

```diff
@@ -94,6 +94,7 @@ void cwbmg( char *, f77int *, f77int * );
 void elemdx( char *, f77int *, f77int );
 void gets1loc( char *, f77int *, f77int *, f77int *, f77int *, f77int );
 f77int ichkstr ( char *, char *, f77int *, f77int, f77int );
+f77int icvidx( f77int *, f77int *, f77int * );
```

Comment:
Should this be
```
f77int icvidx( f77int, f77int, f77int );
```
? The `icvidx` function is defined in `icvidx.c` as:
```
int
icvidx(int ii, int jj, int numjj)
{
   return ( numjj * ii ) + jj;
}
```

Reply:
I checked the icvidx.c. It is defined as:
```
f77int icvidx( f77int *ii, f77int *jj, f77int *numjj )
{
        return ( *numjj * (*ii) ) + *jj;
}
```
I tested it with ```f77int icvidx( f77int, f77int, f77int );```, but it failed.
Therefore, in bufrlib.h it should be
```
f77int icvidx( f77int *, f77int *, f77int * );
```

Reply:
I realize now what happened. I was curious to see if the current NCEPLIBS BUFR library code would be a drop-in replacement, and I had overwritten files in the `var/external/bufr` directory with those from https://github.com/NOAA-EMC/NCEPLIBS-bufr. In that code, `icvidx` is defined differently (and as I had quoted): https://github.com/NOAA-EMC/NCEPLIBS-bufr/blob/develop/src/icvidx.c#L21-L25 .

I think the declaration of `icvidx` in this PR is correct in this case.